lisa/transformers/kernel_source_installer.py

```diff
@@ -155,6 +155,27 @@ def install(self) -> str:
 
         self._install_build_tools(node, runbook.build_deps)
 
+        # Ubuntu sort kernels by version. If installed kernel version is lower than default
+        # one extra steps needed to ensure boot into correct kernel.
+        if isinstance(node.os, Ubuntu):
+            result = node.execute(
+                "cat > /tmp/grub-lisa.cfg <<EOF\n" \
+                "GRUB_DEFAULT=saved\n" \
+                "GRUB_DISABLE_SUBMENU=y\n" \
+                "EOF\n",
+                shell=True
+            )
+            result.assert_exit_code()
+
+            result = node.execute(
```

Comment:
Can it be cat into the `/etc/default/grub.d/99-lisa.cfg` directly? So it doesn't need an extra command to copy.